django/contrib/postgres/indexes.py

```diff
@@ -117,20 +117,27 @@ def get_with_params(self):
 class BTreeIndex(PostgresIndex):
     suffix = "btree"
 
-    def __init__(self, *expressions, fillfactor=None, **kwargs):
+    def __init__(self, *expressions, fillfactor=None, deduplicate_items=None, **kwargs):
         self.fillfactor = fillfactor
+        self.deduplicate_items = deduplicate_items
         super().__init__(*expressions, **kwargs)
 
     def deconstruct(self):
         path, args, kwargs = super().deconstruct()
         if self.fillfactor is not None:
             kwargs["fillfactor"] = self.fillfactor
+        if self.deduplicate_items is not None:
+            kwargs["deduplicate_items"] = self.deduplicate_items
         return path, args, kwargs
 
     def get_with_params(self):
         with_params = []
         if self.fillfactor is not None:
             with_params.append("fillfactor = %d" % self.fillfactor)
+        if self.deduplicate_items is not None:
+            with_params.append(
+                "deduplicate_items = %s" % ("on" if self.deduplicate_items else "off")
+            )
```

Comment:
`deduplicate_items` cannot be used with `include`. Do you think we should raise a `ValueError` in this case?

Reply:
`deduplicate_items` has more limitations, let's leave that to the users.